config/routes.rb

```diff
@@ -7,8 +7,9 @@
   namespace :api do
     namespace :v1 do
       resources :bookings, only: [:index]
-      resource :users, only: [:create]
+      resources :users, only: [:create]
       resources :movies
+      post '/create', to: 'users#create'
```

Comment:
Hello @BouaikIlham 
Good job so far! I would recommend removing the line since you have already defined create action in the resources